src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -3500,6 +3500,9 @@ def create_functionapp(cmd, resource_group_name, name, storage_account, plan=Non
     client = web_client_factory(cmd.cli_ctx)
 
     if vnet or subnet:
+        if environment is not None:
+            raise ArgumentUsageError('vnet paramter and managed environment parameter canot be passed in at the same time', 
+            'This function app is created in an App Environment. Go to App environment to configure Networking')
```

Comment:
Nit (just fixing spelling mistakes in the first line and just changing wording in second line since it currently gives the impression that the function app was created):
```suggestion
            raise ArgumentUsageError('The vnet parameter and managed environment parameter cannot be passed in at the same time', 
            'For managed environment deployments, please configure networking in your container app environment')
```

Reply:
Actually, I just spoke to Ramya and she mentioned that it would be nice if we could run these messages by her and Apeksha. I will be starting a document on this